cmd/log-splicer/main.go

```diff
@@ -1,5 +1,11 @@
 // SPDX-License-Identifier: MIT
 
+// Tasks the log splicer takes care of during its runtime:
+// * Creates folder super structure, encapsulating each identity in its own folder
+// * Maps identities to folder names they live in
+// * Copies secrets from the fixtures folder to each identity folder
+// * Splics out messages from the one ssb-fixtures log.offset to the many log.offset in said folder structure
+// * Persists an identity mapping of ID to {folderName, latest sequence number} to secret-ids.json
```

Comment:
This is a meta comment about using markdown lists, which sadly fails horribly.

here are some options: https://stackoverflow.com/a/52745058

(ps: If you like, you could subscribe to https://github.com/golang/go/issues/7873)

Reply:
> I made a modified version of Godoc that adds ordered and unordered lists with a fast and simple custom parser I wrote called slippery-slope-markdown.

🤣